### PR TITLE
docs: readme corrections from localization

### PR DESCRIPTION
**Pull request by vvtimofeev**
Ok, friends, we plan to add ru loc to gravity-ui.com
So we did some edits in english readme files and added readme in Russian to every component, which present on gravity-ui.com
(Ticket: CLOUDLOC-2378)
If you don't mind, please review your component.

### CODE REVIEW

**Review by goshander — APPROVED** [review]

**Comment by goshander** [comment]
@vvtimofeev preview is not support doc change lang

**Review by mournfulCoroner — APPROVED** [review]

**Review by Estasie — APPROVED** [review]

File: src/components/Pagination/README.md [hunk 1]
```diff
@@ -27,14 +27,14 @@ const pagination = <Pagination page={1} pageSize={100} total={1000} onUpdate={ha
...
 | :-------------- | :------------------------------------------------------------------------------------------------------- | :--------: | :-----: |
-| className       | HTML `class` attribute                                                                                   |  `string`  |         |
+| className       | `class` HTML attribute                                                                                   |  `string`  |         |
 | compact         | Hides the title for the `First`, `Previous`, and `Next` buttons. Always set to `true` in mobile version. | `boolean`  | `true`  |
 | onUpdate        | Called when the page number or `pageSize` is changed                                                     | `Function` |         |
-| size            | Size of the pagination items. By default, 'l' in mobile version, 'm' in desktop version                  |  `string`  |         |
+| size            | Size of the pagination items. By default, its value is `l` in mobile version and `m`, in desktop version |  `string`  |         |
```
Comment (jhoncool) [inline src/components/Pagination/README.md:33]: Is it really correct?
`its value is 'l'`

File: src/components/Palette/loc/README-ru.md [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+| disabled        | Отключает опции.                                                                                          |                       `boolean`                        |        `false`        |
+| multiple        | Включает возможность выбора нескольких опций.                                                             |                       `boolean`                        |        `true`         |
+| onBlur          | Обработчик события `onBlur`.                                                                              | `(event: React.FocusEvent<HTMLButtonElement>) => void` |                       |
+| onFocus         | Обработчик события `onFocus`.                                                                             | `(event: React.FocusEvent<HTMLButtonElement>) => void` |                       |
+| onUpdate        | Срабатывает при изменении состояния пользователем. Передает новое значение как аргумент обратного вызова. |              `(value: string[]) => void`               |                       |
```
Comment (Ruminat) [inline src/components/Palette/loc/README-ru.md:227]: "аргумент обратного вызова" звучит как-то странно. Мне кажется, можно написать просто "Передает новое значение как аргумент"

**Review by Ruminat — APPROVED** [review]

File: src/components/ArrowToggle/loc/README-ru.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
+<!--GITHUB_BLOCK-->
+
+# ArrowToggle
+
+<!--/GITHUB_BLOCK-->
+
+`ArrowToggle` — это компонент для отображения иконки в виде стрелки, которая может вращаться в четырех направлениях. Подходит для отображения выпадающих списков, сокращенных компонентов и других элементов.
```
Comment (Marginy605) [inline src/components/ArrowToggle/loc/README-ru.md:7]: `сокращенных компонентов` doesn't look good. May be use some other phrase?

File: src/components/Table/README.md [hunk 4]
```diff
@@ -8,11 +8,11 @@
 import {Table} from '@gravity-ui/uikit';
 ```
 
-A table that allows the selecting and sorting of rows, and performing actions on a row.
+A `table` allows selecting and sorting rows, as well as performing actions on a row.
```
Comment (Raubzeug) [inline src/components/Table/README.md:11]: `table` -> `Table`

File: src/components/Table/README.md [hunk 5]
```diff
@@ -24,59 +24,58 @@ Additional functionality is enabled via HOCs:
...
+| sticky      | Sticky column                                                                                                      |                     `"start"` `"end"`                      |                                                         |
+| primary     | Identifies a column as primary as opposed to others                                                                |                         `boolean`                          |                                                         |
+| width       | Column's content width in pixels                                                                                   |                     `number` `string`                      |                                                         |
+| meta        | Miscellaneous data including the HOC settings                                                                      |                   `Record<string, any>`                    |                                                         |
 
-## Usage with HOC `withTableActions`
+## Using `table` with the `withTableActions` HOC
```
Comment (Raubzeug) [inline src/components/Table/README.md:68]: `table` -> `Table`

File: src/components/Table/README.md [hunk 6]
```diff
@@ -157,15 +156,15 @@ const table = (
 );
 ```
 
-## Usage with HOC `withTableCopy`
+## Using `table` with the `withTableCopy` HOC
```
Comment (Raubzeug) [inline src/components/Table/README.md:159]: table -> Table

File: src/components/Table/README.md [hunk 7]
```diff
@@ -185,15 +184,15 @@ const columns = [
 const table = <MyTable data={data} columns={columns} />;
 ```
 
-## Usage with HOC `withTableSelection`
+## Using `table` with the `withTableSelection` HOC
```
Comment (Raubzeug) [inline src/components/Table/README.md:187]: table -> Table

File: src/components/Table/README.md [hunk 8]
```diff
@@ -224,9 +223,9 @@ function SelectionTable() {
 }
 ```
 
-## Usage with HOC `withTableSettings`
+## Using `table` with the `withTableSettings` HOC
```
Comment (Raubzeug) [inline src/components/Table/README.md:226]: table -> Table

File: src/components/Table/README.md [hunk 9]
```diff
@@ -331,16 +328,16 @@ function SelectionTable() {
 }
 ```
 
-## Usage with HOC `withTableSorting`
+## Using `table` with the `withTableSorting` HOC
```
Comment (Raubzeug) [inline src/components/Table/README.md:331]: table -> Table

**Review by Raubzeug — APPROVED** [review]

File: src/components/Palette/loc/README-ru.md (hunk 2 above)
Comment (vvtimofeev) [reply src/components/Palette/loc/README-ru.md]: Поправил, спасибо!

File: src/components/Pagination/README.md (hunk 1 above)
Comment (vvtimofeev) [reply src/components/Pagination/README.md:33]: Is this question about the English wording or about the correctness of the default value?

File: src/components/ArrowToggle/loc/README-ru.md (hunk 3 above)
Comment (vvtimofeev) [reply src/components/ArrowToggle/loc/README-ru.md:7]: `компонентов в свёрнутом состоянии` look's good?

File: src/components/Table/README.md (hunk 4 above)
Comment (vvtimofeev) [reply src/components/Table/README.md]: thx, fixed

File: src/components/Table/README.md (hunk 5 above)
Comment (vvtimofeev) [reply src/components/Table/README.md]: thx, fixed

File: src/components/Table/README.md (hunk 6 above)
Comment (vvtimofeev) [reply src/components/Table/README.md]: thx, fixed

File: src/components/Table/README.md (hunk 7 above)
Comment (vvtimofeev) [reply src/components/Table/README.md]: thx, fixed

File: src/components/Table/README.md (hunk 8 above)
Comment (vvtimofeev) [reply src/components/Table/README.md]: thx, fixed

File: src/components/Table/README.md (hunk 9 above)
Comment (vvtimofeev) [reply src/components/Table/README.md]: thx, fixed

File: src/components/PlaceholderContainer/README.md [hunk 10]
```diff
@@ -4,19 +4,19 @@
 
 <!--/GITHUB_BLOCK-->
 
-`PlaceholderContainer` is a component for displaying content with image, text content and action controls.
+`PlaceholderContainer` is a component for displaying content with images, text content, and action controls.
```
Comment (Marginy605) [inline src/components/PlaceholderContainer/README.md:7]: Why images in plural?

File: src/components/PlaceholderContainer/README.md [hunk 11]
```diff
@@ -162,7 +162,7 @@ with src and alt settings
 
 <!--/GITHUB_BLOCK-->
 
-The content of component contains from title and description blocks that can be set by the same properties names. To render custom content use `content` property.
+The `PlaceholderContainer` content is borrowed from the title and description sections, which you can set with the appropriate properties. To render custom content, use the `content` property.
```
Comment (Marginy605) [inline src/components/PlaceholderContainer/README.md:165]: 'is borrowed' breaks the meaning of sentence

File: src/components/PlaceholderContainer/loc/README-ru.md [hunk 12]
```diff
@@ -0,0 +1,221 @@
+<!--GITHUB_BLOCK-->
+
+# PlaceholderContainer
+
+<!--/GITHUB_BLOCK-->
+
+`PlaceholderContainer` — это компонент для отображения контента с изображениями, текстовой информацией и контролами действий.
```
Comment (Marginy605) [inline src/components/PlaceholderContainer/loc/README-ru.md:7]: 'с изображениями'  ->  'с иллюстрацией'

File: src/components/PlaceholderContainer/loc/README-ru.md [hunk 13]
```diff
@@ -0,0 +1,221 @@
...
+  image={{
+    src: 'https://gravity-ui.com/static/images/design/Resources/img-gravitylib.png',
+    alt: 'logo',
+  }}
+/>
+```
+
+<!--/GITHUB_BLOCK-->
+
+Контент `PlaceholderContainer` берется из секций заголовка и описания, которые задаются через соответствующие свойства. Для отображения пользовательского контента используйте свойство `content`.
```
Comment (Marginy605) [inline src/components/PlaceholderContainer/loc/README-ru.md:165]: 'берется' -> 'состоит'

File: src/components/PlaceholderContainer/loc/README-ru.md [hunk 14]
```diff
@@ -0,0 +1,221 @@
...
+| align       | Задает горизонтальное выравнивание контента. Возможные значения: `"center"` и `"left"`. |                                   `string`                                    |      `"center"`       |
+| title       | Текст заголовка контента.                                                               |                                   `string`                                    |                       |
+| description | Текст описания контента.                                                                |                                   `string`                                    |                       |
+| image       | Задает изображение через `src` или передает узел React.                                 | `PlaceholderContainerImageNodeProps`<br/> `\| PlaceholderContainerImageProps` |                       |
+| content     | Используется для рендеринга узла вместо контента (заголовка, описания и действий).      |                               `React.ReactNode`                               |                       |
```
Comment (Marginy605) [inline src/components/PlaceholderContainer/loc/README-ru.md:220]: 'узла' sounds unnatural

File: src/components/ArrowToggle/loc/README-ru.md (hunk 3 above)
Comment (vvtimofeev) [reply src/components/ArrowToggle/loc/README-ru.md]: Thx, fixed.

File: src/components/PlaceholderContainer/loc/README-ru.md (hunk 12 above)
Comment (vvtimofeev) [reply src/components/PlaceholderContainer/loc/README-ru.md:7]: fixed

File: src/components/PlaceholderContainer/loc/README-ru.md (hunk 13 above)
Comment (vvtimofeev) [reply src/components/PlaceholderContainer/loc/README-ru.md:165]: fixed

File: src/components/PlaceholderContainer/loc/README-ru.md (hunk 14 above)
Comment (vvtimofeev) [reply src/components/PlaceholderContainer/loc/README-ru.md]: fixed

File: src/components/PlaceholderContainer/README.md (hunk 10 above)
Comment (vvtimofeev) [reply src/components/PlaceholderContainer/README.md:7]: Got a comment from loc team: "cause if you use a countable noun, you either use an article or a plural form. I was using my best guess here. if there may only be one image, use an image."

Changed to "an image".

File: src/components/PlaceholderContainer/README.md (hunk 11 above)
Comment (vvtimofeev) [reply src/components/PlaceholderContainer/README.md:165]: reverted to 'contains'

**Review by Marginy605 — APPROVED** [review]

**Review by SeqviriouM — APPROVED** [review]

**Review by ValeraS — CHANGES_REQUESTED** [review]

File: src/components/lab/Breadcrumbs/README.md [hunk 15]
```diff
@@ -336,8 +336,8 @@ LANDING_BLOCK-->
 
 ### Integration with routers
 
-`Breadcrumbs` component accepts navigate function received from your router for performing a client side navigation programmatically.
-The following example shows the general pattern.
+The `Breadcrumbs` component accepts the navigation function received from your router for managing the client-side navigation in a software-based mode.
```
[the remark this answers is not in the document]
Comment (vvtimofeev) [reply src/components/lab/Breadcrumbs/README.md:339]: It means that some programs on the user side decide how to display breadcrumbs.
Are you sure that the original is correct?

programmatic | ˌprəʊɡrəˈmatɪk |
adjective
1 of the nature of or according to a programme, schedule, or method: a programmatic approach to change.
2 of the nature of programme music: programmatic tone poems and operas.

I can't get the idea. If you are sure, I will revert changes.

**Review by NikitaCG — APPROVED** [review]

File: src/components/Pagination/README.md (hunk 1 above)
Comment (jhoncool) [reply src/components/Pagination/README.md:33]: first

Comment (vvtimofeev) [reply src/components/Pagination/README.md:33]: I rechecked it with the translator and he said "Yes, it is".

**Review by jhoncool — APPROVED** [review]

File: src/components/lab/Breadcrumbs/README.md (hunk 15 above)
[the remark this answers is not in the document]
Comment (vvtimofeev) [reply src/components/lab/Breadcrumbs/README.md:339]: @ValeraS please, find some time to respond.

**Comment by vvtimofeev** [comment]
@amje @ogonkov @IsaevAlexandr @sofiushko @zamkovskaya @Arucard89 @korvin89 @DakEnviy @kseniya57 @Vladeeg @d3m1d0v @axtk @v4dyar4 if you don't mind, please find some time, to review my PR.

File: src/components/Slider/README.md [hunk 16]
```diff
@@ -190,143 +190,96 @@ LANDING_BLOCK-->
...
 LANDING_BLOCK-->
 
 <!--GITHUB_BLOCK-->
 
 ```tsx
-<Slider tooltipDisplay="on" />
+<Slider hasTooltip={true} />
 ```
 
 <!--/GITHUB_BLOCK-->
```
Comment (Arucard89) [inline src/components/Slider/README.md:260]: Check Slider's description one more time please. You translated an old version. 
Now `hasTooltip`, `availableValues` and `marksCount` are deprecated. 
So the table with properties is also has been changed.

File: src/components/Slider/README.md [hunk 17]
```diff
@@ -8,13 +8,13 @@
 import {Slider} from '@gravity-ui/uikit';
 ```
 
-Slider is a customizable and responsive React component that allows users to select a single value or a range of values from a specified data set.
+A slider is a customizable and responsive React component that allows users to select a single value or a range of values from a specified data set.
```
Comment (Arucard89) [inline src/components/Slider/README.md:11]: It's not about a slider in common. This description for the Slider from uikit library. 
may be `The Slider` will be correct? (like in ru readme)

File: src/components/Slider/README-ru.md [hunk 18]
```diff
@@ -0,0 +1,285 @@
...
+| [step](#step)                        | Шаги приращения и уменьшения при перемещении слайдера. Свойство игнорируется, если задано `availablevalues`.                                                           |                       `number`                       |          `1`          |
+| tabIndex                             | Атрибут `tabIndex` для контрола.                                                                                                                                       |             `number` `[number, number]`              |                       |
+| [validationState](#error)            | Состояние валидации.                                                                                                                                                   |                     `"invalid"`                      |                       |
+| [value](#slider-variants)            | Значение контрола.                                                                                                                                                     |             `number` `[number, number]`              |                       |
```
Comment (Arucard89) [inline src/components/Slider/README-ru.md:285]: The same comment as for en version. We have newer version of readme(with new API) than the translated.

**Review by Arucard89 — CHANGES_REQUESTED** [review]

File: src/components/Slider/README.md (hunk 17 above)
Comment (vvtimofeev) [reply src/components/Slider/README.md:11]: Sorry for the delay, i will come back as soon, as i get the new version of the translation.

File: src/components/Slider/README.md (hunk 16 above)
Comment (vvtimofeev) [reply src/components/Slider/README.md]: Hi, fixed, please recheck!

File: src/components/Slider/README.md (hunk 17 above)
Comment (vvtimofeev) [reply src/components/Slider/README.md:11]: Hi, fixed, please recheck!

File: src/components/Slider/README-ru.md (hunk 18 above)
Comment (vvtimofeev) [reply src/components/Slider/README-ru.md]: Hi, fixed, please recheck!

**Review by amje — APPROVED** [review]

**Review by Arucard89 — APPROVED** [review]

File: src/components/lab/Breadcrumbs/README.md (hunk 15 above)
[the remark this answers is not in the document]
Comment (ValeraS) [reply src/components/lab/Breadcrumbs/README.md:339]: let's revert for now, this part of the readme will be removed in other PR.

File: src/components/lab/Breadcrumbs/loc/README-ru.md [hunk 19]
```diff
@@ -0,0 +1,501 @@
...
+
+<!-- Storybook example -->
+
+<BreadcrumbsWithIcons />
+
+<!--/GITHUB_BLOCK-->
+
+### Интеграция с роутерами
+
+Компонент `Breadcrumbs` принимает функцию навигации от роутера для управления навигацией на стороне клиента в программном режиме.
```
[the remark this answers is not in the document]
Comment (vvtimofeev) [reply src/components/lab/Breadcrumbs/loc/README-ru.md]: Applied your suggestion, thx.

File: src/components/lab/Breadcrumbs/README.md (hunk 15 above)
[the remark this answers is not in the document]
Comment (vvtimofeev) [reply src/components/lab/Breadcrumbs/README.md]: Reverted. Pls, check that it is good now.

**Review by ValeraS — APPROVED** [review]

**Comment by vvtimofeev** [comment]
@amje rebased branch and fixed conflicts, if you don't mind, please merge it.

**Pull request merged**